Class/CleanupVMs.ps1

```diff
@@ -5,66 +5,99 @@ param
     [string] $LabName,
 
     # Credential path
-    [Parameter(Mandatory=$false, HelpMessage="Path to file with Azure credentials")]
-    [string] $credentialPath = "$env:APPDATA\AzProfile.txt"
+    [Parameter(Mandatory=$false, HelpMessage="Path to file with Azure Profile")]
+    [string] $profilePath = "$env:APPDATA\AzProfile.txt"
 )
 
 # Stops at the first error instead of continuing and potentially messing up things
-$global:erroractionpreference = 1
+#$global:erroractionpreference = 1
```

Comment:
yes, capturing all errors was my intent, however, I'm not sure if how I've implemented if is the right way to do that. Investigating alternatives now.